tasmota/xsns_62_MI_HM10.ino

```diff
@@ -1113,23 +1202,16 @@ void HM10Show(bool json)
           if(MIBLEsensors.at(i).lux!=0x0ffffff){ // this is the error code -> no temperature
             ResponseAppend_P(PSTR(",\"" D_JSON_ILLUMINANCE "\":%s"), lux);
           }
-          if(MIBLEsensors.at(i).moisture!=-1000.0f){ // this is the error code -> no moisture
+          if(!isnan(MIBLEsensors.at(i).moisture)){ // this is the error code -> no moisture
```

Comment:
It's better to use `isnanf()` rather than `isnan()`. It will save you a conversion from `float` to `double` and some decent amount of code.

Reply:
I fear, that is no option:

PlatformIO/Projects/Tasmota/tasmota/xsns_62_MI_HM10.ino:1205:49: error: 'isnanf' was not declared in this scope
           if(!isnanf(MIBLEsensors.at(i).moisture)){ // this is the error code -> no moisture

Reply:
I believe, there is no conversion:

tools/sdk/libc/xtensa-lx106-elf/include/math.h:
```
#ifndef isnan
  #define isnan(y) (fpclassify(y) == FP_NAN)
#endif
```

......
Arduino/math.h:
```
#define fpclassify(__x) \
	((sizeof(__x) == sizeof(float))  ? __fpclassifyf(__x) : \
	__fpclassifyd(__x))
```